GamePortal/AliaksNad.Battleship.Data/Contexts/UsersContexts.cs

```diff
@@ -34,4 +30,24 @@ protected override void OnModelCreating(DbModelBuilder modelBuilder)
             modelBuilder.Configurations.Add(new UserDbConfiguration());
         }
     }
+
+    public sealed class FleetContexts : DbContext
+    {
+        public FleetContexts(ILogger logger)
+        {
+            //Database.SetInitializer<UsersContexts>(new MigrateDatabaseToLatestVersion<UsersContexts, Configuration>());
+            Database.SetInitializer<FleetContexts>(new DropCreateDatabaseAlways<FleetContexts>());
```

Comment:
are you sure?

Reply:
yes, right now i develop db and don't need to keep data